packages/proposals/src/actions/registerProgramGovernance.ts

```diff
@@ -111,11 +112,15 @@ export const registerProgramGovernance = async (
     );
   }
 
+  const council_mint_seed = uninitializedTimelockConfig.councilMint
```

Comment:
camel case in JS land, please :)

Reply:
He, he, I've been brainwashed by Rust :), fixed